ui/src/store/serializers.ts

```diff
@@ -29,36 +27,10 @@ const displayLanguage = ['en', 'de', 'fr', '']
 export function serializeProjectsCollection (collection: ProjectsCollection): ProjectsCollection {
   return Object.freeze({
     ...serializeResource(collection),
-    member: collection.member.map(serializeProject),
+    member: collection.member,
```

Comment:
Any reason for this change? It breaks the projects list

Reply:
That's a bit of an overstatement. You mean the organisation name does not show?

Reply:
Well... yeah, it breaks it ;)